src/typescript-reporter/reporter/TypeScriptReporter.ts

```diff
@@ -178,6 +178,19 @@ function createTypeScriptReporter(configuration: TypeScriptReporterConfiguration
     return dependencies;
   }
 
+  function getArtifacts(): FilesMatch {
+    parsedConfiguration = parseConfigurationIfNeeded();
+
+    const [parseConfigFileHost] = getParseConfigFileHost();
+
+    return getArtifactsFromTypeScriptConfiguration(
+      typescript,
+      parsedConfiguration,
+      configuration.context,
+      parseConfigFileHost
+    );
+  }
```

Comment:
maybe we should cache it to avoid parsing all config files for each compilation...